src/components/EventDetail.tsx

```diff
@@ -114,10 +115,21 @@ const EventDetail: FunctionComponent<CalendarUIProps> = ({
           )}
           {open && (
             <Button
-              variant="contained"
-              style={{ marginBottom: 30, alignSelf: "center" }}
+              key="MakeBooking"
+              style={{
+                backgroundColor: "Green",
+                color: "white",
+                maxWidth: "400px",
+              }}
+              onClick={(event): void => {
+                event.stopPropagation();
```

Comment:
Just curious about this `stopPropagation` - is there another listening that is getting trigger on click?